README.md

```diff
@@ -254,6 +254,19 @@ We are super excited about our first-ever Ultralytics YOLOv5 🚀 EXPORT Competi
 
 </details>
 
+### Classification Checkpoints
+
+| Model                                                                                              | size<br><sup>(pixels) | accuracy<br><sup>top1 | accuracy<br><sup>top5 | Train time<br><sup>90 epochs<br>(hours) | Speed<br><sup>CPU<br>(ms) | Speed<br><sup>V100<br>(ms) | params<br><sup>(M) | FLOPs<br><sup>@640 (B) |
+|----------------------------------------------------------------------------------------------------|-----------------------|-----------------------|-----------------------|-----------------------------------------|---------------------------|----------------------------|--------------------|------------------------|
+| [ResNet50](https://github.com/ultralytics/yolov5/releases/download/v6.2/resnet50.pt)               | 224                   | 76.8                  | 93.4                  | 11:10                                   | 61.8                      | 9.2                        | 25.6               | 8.3                    |
+| [EfficientNet_b0](https://github.com/ultralytics/yolov5/releases/download/v6.2/efficientnet_b0.pt) | 224                   | 75.1                  | 92.4                  | 13:03                                   | 31.8                      | 11.2                       | 5.3                | 0.8                    |
+| [YOLOv5n-cls](https://github.com/ultralytics/yolov5/releases/download/v6.2/yolov5n-cls.pt)         | 224                   | 64.6                  | 85.4                  | 7:59                                    | 8.0                       | 3.9                        | 2.5                | 0.4                    |
+| [YOLOv5s-cls](https://github.com/ultralytics/yolov5/releases/download/v6.2/yolov5s-cls.pt)         | 224                   | 71.5                  | 90.2                  | 8:09                                    | 14.5                      | 4.6                        | 5.5                | 1.3                    |
+| [YOLOv5m-cls](https://github.com/ultralytics/yolov5/releases/download/v6.2/yolov5m-cls.pt)         | 224                   | 75.9                  | 92.9                  | 10:06                                   | 30.1                      | 6.3                        | 13.0               | 3.8                    |
+| [YOLOv5l-cls](https://github.com/ultralytics/yolov5/releases/download/v6.2/yolov5l-cls.pt)         | 224                   | 78.0                  | 94.0                  | 11:56                                   | 55.4                      | 8.3                        | 26.6               | 8.4                    |
+| [YOLOv5x-cls](https://github.com/ultralytics/yolov5/releases/download/v6.2/yolov5x-cls.pt)         | 224                   | 78.9                  | 94.4                  | 15:04                                   | 96.5                      | 10.1                       | 48.1               | 15.9                   |
+
+
```

Comment:
@glenn-jocher I think you're already planning to make this section collapsible right?
How about this structure instead:
- [head1] Classification
    Example command for classification training and inference
   - [Collapsible head2] Classification Checkpoints

Reply:
Yes, the current README content needs updating. I'll work on that a little later. I've refactored classification into /classify now, will add classification CI today.

<img width="596" alt="Screenshot 2022-08-14 at 13 29 38" src="https://user-images.githubusercontent.com/26833433/184534649-a39847da-4354-4210-b83b-1d4f2ec66c8a.png">